kylecornelissen/vacation-rentals
# Add listing card info

What's this PR do?
- Added images for all listings to public folder (`public/images`)
- Added image to each listing card in the ListingContainer
- Added a button to each listing card that links to an Individual Listing
- Added the beginnings of a Listing component
- Added a route in App.js to show Listing
- Removed obsolete imports and added a black background to `<body>` styling

How should this be manually tested?

- npm start
- navigate to localhost:3000/areas/ and click on an area to see the listings

What are the relevant tickets?
#12 

Future Implementation Plans:
- Add full Listing page with additional info
